index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portfolio</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="nav-container">
+        <div class="time-network">
+          <h4>9:41</h4>
+          <ul class="network-only">
+            <li class="network-only-item">
+              <img src="image/CellularConnection.png" alt="" />
+            </li>
+            <li class="network-only-item">
+              <img src="image/Wifi.svg" alt="" />
+            </li>
+            <li class="network-only-item">
+              <img src="image/Capacity.svg" alt="" />
+            </li>
+          </ul>
+        </div>
+        <div class="name-and-burger">
+          <h2>Yohannes Dag</h2>
```

Comment:
- [ ] The logo on the left should have a link (`a tag`). Please work on it. :writing_hand:

Reply:
Kindly implement all changes the previous code reviewer asked for

Reply:
- [ ] Kindly replace the `h2` tag with `a` tag.